data_validation/cli_tools.py

```diff
@@ -585,6 +585,7 @@ def _configure_custom_query_parser(custom_query_parser):
         "--primary-keys",
         "-pk",
         help="Comma separated list of primary key columns 'col_a,col_b'",
+        required=True,
```

Comment:
The issue with adding required here is that custom query column parser doesn't require a PK. PR #673 is doing something similar